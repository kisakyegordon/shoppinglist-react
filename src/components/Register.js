import React, {Component} from 'react';
import { PropTypes } from 'prop-types';
import RegisterForm from '../components/forms/registerform';
import { connect } from 'react-redux';

class Register extends Component{
    render(){

        return (
            
            <div>
            <div className="signin">
            <h1> Register </h1>
            <RegisterForm />
            </div>
            </div>
        );
    }
}

export default Register;


