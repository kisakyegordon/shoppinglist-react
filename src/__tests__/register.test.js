import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import Register from '../components/register/register';

describe('<Register />', () => {
    it('match a snapshot', () => {
        const wrapper = shallow(<Register />);
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });

});