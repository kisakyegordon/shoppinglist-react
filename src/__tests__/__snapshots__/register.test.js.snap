// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Register /> match a snapshot 1`] = `
<div>
  <div
    className="signin"
  >
    <h1>
       Register 
    </h1>
    <RegisterForm />
  </div>
</div>
`;
